.env.example

```
@@ -0,0 +1,22 @@
+# Sei Network Configuration
+SEI_RPC_URL=https://evm-rpc.sei-apis.com
+SEI_CHAIN_ID=1329
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The SEI_CHAIN_ID key should go before the SEI_RPC_URL key

(UnorderedKey)